metadata/db/db.go

```diff
@@ -357,23 +352,14 @@ func writeMetadataEntry(md *bolt.Bucket, m *metadataEntry) error {
 	if err := putFileSize(md, bucketKeyUncompressedOffset, m.UncompressedOffset); err != nil {
 		return errors.Wrapf(err, "failed to set UncompressedOffset value %d", m.UncompressedOffset)
 	}
-	if err := putSpanID(md, bucketKeySpanStart, m.SpanStart); err != nil {
-		return errors.Wrapf(err, "failed to set SpanStart value %d", m.SpanStart)
-	}
-	if err := putSpanID(md, bucketKeySpanEnd, m.SpanEnd); err != nil {
-		return errors.Wrapf(err, "failed to set SpanEnd value %d", m.SpanEnd)
-	}
+
 	return nil
 }
 
 func putFileSize(b *bolt.Bucket, k []byte, v soci.FileSize) error {
 	return putInt(b, k, int64(v))
 }
 
-func putSpanID(b *bolt.Bucket, k []byte, v soci.SpanID) error {
```

Comment:
Just curious how does this effect SpanID?  does this change how we use it?

Reply:
It does not, we just do not store spanStart and spanEnd in metadata db. Since we anyway deal with spans in other parts of the snapshotter, we still need to keep the type `SpanID`.